src/index.css

```diff
@@ -1,13 +1,87 @@
+@tailwind base; /* stylelint-disable-line */
+@tailwind components; /* stylelint-disable-line */
+@tailwind utilities; /* stylelint-disable-line */
```

Comment:
Let's remove all disabled Linters, Linting helps achieve cleaner and professional code ✊

Kindly check for other occurrences 🙂

Reply:
Tailwind isn't recognized by linters. it can only work with linters disabled